README.md

```diff
@@ -78,25 +84,33 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+app.get("/", function(request, response){
+  response.send("Index");
+});
+app.post("/", function(request, response){
+  response.send("Create");
+});
+app.put("/:id", function(request, response){
+  response.send("Update");
+});
+app.delete("/:id", function(request, response){
+  response.send("Delete");
+});
 
 ```
 
-```js
-// Your answer...
-```
 ### Question #7
 
 Describe the differences between Express and Rails as backend frameworks.
 
 ```text
-
+Rails is heavily structured and requires a you to write code in a more specific way.  Express gives you much more freedom when it comes to how you wish to structure your app.
 ```
 
 ### Question #8
 
-What is the importance of using body-parser in our express application for post requests? 
+What is the importance of using body-parser in our express application for post requests?
 
 ```js
-
+Body-parser must be used to process user input from forms in order to make a post request.
 ```
```

Comment:
👍  body parser allows us to parse the body of any post request and we use it to access user input via form data